src/components/contact-center/members/opened-queue-member-general.vue

```diff
@@ -75,7 +85,7 @@
                 set(value) {this.setItemProp({prop: 'priority', value})}
             },
             expireAt: {
-                get() {return this.$store.state.ccenter.queues.members.itemInstance.expireAt},
+                get() {return this.$store.state.ccenter.queues.members.itemInstance.expireAt || 0},
```

Comment:
default values should be set on api response handling stage, or, at least, in store -- there's no need to set them in ui